app/controllers/FeedbackFormsController.scala

```diff
@@ -140,6 +142,22 @@ class FeedbackFormsController @Inject()(val messagesApi: MessagesApi,
     }
   }
 
+  def getFeedbackFormPreview: Action[JsValue] = AdminAction.async(parse.json) { implicit request =>
+    (request.body \ "id").asOpt[String].fold {
+      Logger.error(s"Received a bad request form id to update not found")
+      Future.successful(BadRequest("Malformed data!"))
+    } { id =>
+      feedbackRepository
+        .getByFeedbackFormId(id)
+        .map {
+          case Some(feedbackForm: FeedbackForm) =>
+            val json = s"""{"status":${Json.toJson(feedbackForm).toString}}"""
```

Comment:
Can we not use model case class directly in views ? Create a new case class for using in views with only those parameters that are required on the view.